deploy/olm-catalog/service-binding-operator/service-binding-operator.package.yaml

```diff
@@ -1,5 +1,7 @@
 channels:
 - currentCSV: service-binding-operator.vBUNDLE_VERSION
+  name: alpha-0.2
+- currentCSV: service-binding-operator.v0.1.1-364
   name: alpha
-defaultChannel: alpha
+defaultChannel: alpha-0.2
```

Comment:
default channel should remain alpha, because we do not want to break existing OCP installations.

Reply:
Done. ce8ced142aed6e54659609cf8f25031ef4a4c575